test/functional/feature_utxo_set_hash.py

```diff
@@ -58,7 +56,7 @@ def test_muhash_implementation(self):
                         continue
 
                     data = COutPoint(int(tx.rehash(), 16), n).serialize()
-                    data += struct.pack("<i", height * 2 + coinbase)
```

Comment:
if possible, can you cover it with a test that reproduces the error before this fix - and passes after it?